ts/@live-compositor/core/src/compositorManager.ts

```diff
@@ -16,3 +16,7 @@ export interface CompositorManager {
   registerEventListener(cb: (event: unknown) => void): void;
   terminate(): Promise<void>;
 }
+
+export interface NodeCompositorManager extends CompositorManager {
+  sendMultipartRequest(request: MultipartRequest): Promise<object>;
```

Comment:
after thinking it through I think you should move that method to CompositorManager and remove that new manager

For `web-wasm` just implement empty method that throws an error.